Orange/widgets/evaluate/tests/test_owconfusionmatrix.py

```diff
@@ -40,19 +46,11 @@ def test_selected_learner(self):
         self.send_signal("Evaluation Results", self.results_1_iris)
         self.widget.selected_learner[:] = [0]
 
-    def test_outputs(self):
-        self.send_signal("Evaluation Results", self.results_1_iris)
-
-        # check selected data output
-        self.assertIsNone(self.get_output("Selected Data"))
-
-        # select data instances
+    def _select_data(self):
         self.widget.select_correct()
-
-        # check selected data output
-        selected = self.get_output("Selected Data")
-        self.assertGreater(len(selected), 0)
-
-        # check output when data is removed
-        self.send_signal("Evaluation Results", None)
-        self.assertIsNone(self.get_output("Selected Data"))
+        indices = self.widget.tableview.selectedIndexes()
+        indices = {(ind.row() - 2, ind.column() - 2) for ind in indices}
+        selected = [i for i, t in enumerate(zip(
+            self.widget.results.actual, self.widget.results.predicted[0]))
+                    if t in indices]
+        self.selected_indices = self.widget.results.row_indices[selected]
```

Comment:
Would it be nicer if `_select_data` returned `selected_indices` instead of (ab?)using the instance's attributes for semi-global data storage?